net/openssh/files/sshd.init

```diff
@@ -6,8 +6,99 @@ STOP=50
 
 USE_PROCD=1
 PROG=/usr/sbin/sshd
+NAME=sshd
 
-start_service() {
+. /lib/functions.sh
+. /lib/functions/network.sh
+
+validate_section_sshd()
+{
+	uci_load_validate sshd sshd "$1" "$2" \
+		'PasswordAuth:bool:1' \
+		'RootPasswordAuth:bool:1' \
+		'RootLogin:bool:1' \
+		'Interface:string' \
+		'Port:port:22' \
+		'IdleTimeout:uinteger:0' \
+		'MaxAuthTries:uinteger:3' \
+		'enable:bool:1' \
+		'mdns:bool:1'
+}
+
+# because sshd does not have an option for specifying an interface
+# but only for specifying listen address
+# we get the addresses of interface and add them
+append_addresses()
+{
+	local ipaddrs="${1}"
+	local port="${2}"
+
+	procd_append_param command -o "Port ${port}"
+
+	for addr in $ipaddrs; do
+		procd_append_param command -o "ListenAddress ${addr}"
+	done
+}
+
+set_params()
+{
+	append_addresses "${ipaddrs}" "${Port}"
+
+	[ "${PasswordAuth}" -eq 0 ] && procd_append_param command -o "PasswordAuthentication no"
+	[ "${RootPasswordAuth}" -eq 1 ] && procd_append_param command -o "PermitRootLogin yes"
+	[ "${RootLogin}" -eq 0 ] && procd_append_param command -o "PermitRootLogin no"
+	[ "${MaxAuthTries}" -gt 0 ] && procd_append_param command -o "MaxAuthTries ${MaxAuthTries}"
+	[ "${mdns}" -ne 0 ] && procd_add_mdns "ssh" "tcp" "${Port}" "daemon=sshd"
+
+	if [ "${IdleTimeout}" -ne 0 ]; then
+		procd_append_param command -o "ClientAliveCountMax 1"
```

Comment:
`ClientAliveCountMax 1` may be too small.
By default it's 3 so maybe keep it as is?

https://manpages.debian.org/stretch/openssh-server/sshd_config.5.en.html#ClientAliveCountMax

Reply:
This is my logic:

from https://www.man7.org/linux/man-pages/man5/sshd_config.5.html

**ClientAliveCountMax:**
The default value is 3.  If ClientAliveInterval is set to
15, and ClientAliveCountMax is left at the default,
unresponsive SSH clients will be disconnected after
approximately 45 seconds.  Setting a zero
ClientAliveCountMax disables connection termination.

Therefore, to mimic IdleTimeout from dropbear, we set
ClientAliveCountMax to 1 and ClientAliveInterval to
the time we want to wait.